.github/workflows/cleanup.yml

```diff
@@ -65,4 +65,32 @@ jobs:
               --name "$account" \
               --resource-group "$RESOURCE_GROUP" \
               --yes
+          done
+
+  cleanup-akv:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Log into Azure
+        uses: azure/login@v2
+        with:
+          client-id: ${{ secrets.MANAGED_ID_CLIENT_ID }}
+          tenant-id: ${{ secrets.MANAGED_ID_TENANT_ID }}
+          subscription-id: 7ca35580-fc67-469c-91a7-68b38569ca6e
+
+      - run: |
+          source services/cacitesting.env
+          key_vaults=$(az keyvault list \
+            --resource-group "$RESOURCE_GROUP" \
```

Comment:
we should use a specific KV name for our tests to avoid deleting of existing keys

Reply:
In the resource group in question, we only deploy AKV instances for tests, so it's fine to delete everything. This exact logic applies to the container instances and storage accounts in the jobs above this one

Reply:
so cleanup will only be scheduled?